src/data/localization.rs

```diff
@@ -285,6 +285,41 @@ impl Localization {
         }
     }
 
+    #[cfg(feature = "ck3")]
+    pub fn verify_name_exists(&self, name: &Token, max_sev: Severity) {
+        if name.as_str().is_empty() {
+            return;
+        }
+        self.mark_used(name.as_str());
+        let mut langs: Vec<&str> = Vec::new();
+        for lang in &self.mod_langs {
+            let hash = self.locas.get(lang);
+            if hash.is_none() || !hash.unwrap().contains_key(name.as_str()) {
+                langs.push(lang);
+            }
+        }
+        if !langs.is_empty() {
+            // It's merely untidy if the name is only missing in latin-script languages and the
+            // name doesn't have indicators that it really needs to be localized (such as underscores
+            // or extra uppercase letters). In all other cases it's a warning.
+            let sev = if only_latin_script(&langs)
+                && !name.as_str().contains('_')
+                && name.as_str().chars().filter(|c| c.is_uppercase()).count() == 1
```

Comment:
Names with multiple parts may contain more than one uppercase chars

Reply:
Yeah but in practice those (should) contain _ so that they're localization. Having a space in a name is actually a greater sin because then you can't localize it at all :)

Reply:
Unfortunately, there are names connected with hyphen, like "Al-Nasir"